editor/src/components/menubar/menubar.tsx

```diff
@@ -119,6 +119,7 @@ export const Menubar = betterReactMemo('Menubar', () => {
   )
 
   const onDoubleClick = React.useCallback(() => {
+    // TODO KILL ME
```

Comment:
Follow up work?

Reply:
@OmarDaSilva as I mentioned, this _entire_ callback isn't necessary anymore and can be removed

Reply:
Sorted